<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBoxStartup">
         <property name="title">
          <string>Startup</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="startMinimizedCheckBox">
            <property name="text">
             <string>Start minimized to system tray</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="autoConnectCheckBox">
            <property name="text">
             <string>Auto-connect to last server</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkUpdatesCheckBox">
            <property name="text">
             <string>Check for updates on startup</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxDisplay">
         <property name="title">
          <string>Display</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelDefaultColorDepth">
            <property name="text">
             <string>Default Color Depth:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="defaultColorDepthComboBox">
            <item>
             <property name="text">
              <string>16-bit (High Color)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24-bit (True Color)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32-bit (True Color + Alpha)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelDefaultResolution">
            <property name="text">
             <string>Default Resolution:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="defaultResolutionComboBox">
            <item>
             <property name="text">
              <string>1024x768</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1280x720</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1280x1024</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1366x768</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1440x900</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1600x900</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1920x1080</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSecurity">
      <attribute name="title">
       <string>Security</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBoxEncryption">
         <property name="title">
          <string>Encryption</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="enableEncryptionCheckBox">
            <property name="text">
             <string>Enable encryption</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="requireEncryptionCheckBox">
            <property name="text">
             <string>Require encryption</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="enableNLA">
            <property name="text">
             <string>Enable Network Level Authentication (NLA)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxCertificates">
         <property name="title">
          <string>Certificates</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="verifyCertificatesCheckBox">
            <property name="text">
             <string>Verify server certificates</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="warnOnCertMismatchCheckBox">
            <property name="text">
             <string>Warn on certificate mismatch</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPerformance">
      <attribute name="title">
       <string>Performance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBoxOptimization">
         <property name="title">
          <string>Optimization</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QCheckBox" name="enableHardwareAccelerationCheckBox">
            <property name="text">
             <string>Enable hardware acceleration</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="enableBitmapCachingCheckBox">
            <property name="text">
             <string>Enable bitmap caching</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="enableCompressionCheckBox">
            <property name="text">
             <string>Enable compression</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxQuality">
         <property name="title">
          <string>Quality Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelImageQuality">
            <property name="text">
             <string>Image Quality:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="imageQualityComboBox">
            <item>
             <property name="text">
              <string>Low (Best Performance)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Medium (Balanced)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>High (Best Quality)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset to Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 